protocol/rpcconsumer/rpcconsumer.go

```diff
@@ -170,12 +171,20 @@ func (rpcc *RPCConsumer) Start(ctx context.Context, txFactory tx.Factory, client
 				return err
 			}
 			chainID := rpcEndpoint.ChainID
+			// create policyUpdaters per chain
+			if policyUpdater, ok := policyUpdaters[rpcEndpoint.ChainID]; ok {
+				policyUpdater.AddPolicySetter(chainParser, *rpcEndpoint)
```

Comment:
we are ignoring the error here, but this error will mean we aren't setting the policy updater, so let's either make it fatal, or overwrite the policy updater